include/dt-bindings/clock/imx_ccm.h

```diff
@@ -20,4 +20,21 @@
 #define IMX_CCM_CAN_CLK			10
 #define IMX_CCM_GPT_CLK			11
 
+#define IMX_CCM_SAI1_CLK        12
+#define IMX_CCM_SAI2_CLK        13
+#define IMX_CCM_SAI3_CLK        14
+
+#define IMX_CCM_CS1CDR_PREDIV   30
+#define IMX_CCM_CS1CDR_PODF     31
+#define IMX_CCM_CS2CDR_PREDIV   32
+#define IMX_CCM_CS2CDR_PODF     33
+
+#define IMX_CCM_AUDIO_BYPASS_SRC 40
+#define IMX_CCM_AUDIO_DIV_SEL    41
+#define IMX_CCM_AUDIO_POST_DIV_SEL 42
+
+#define IMX_CCM_ANALOG_PLL_AUDIO_NUM 50
+#define IMX_CCM_ANALOG_PLL_AUDIO_DENOM 51
+
```

Comment:
Why are these defines needed as these values are not providing any information

Reply:
this is reqired by DTS, see below
```
pll-clocks = <&anatop IMX_CCM_AUDIO_BYPASS_SRC 0x70 0xC000 0
				   &anatop IMX_CCM_AUDIO_DIV_SEL 0x70 0x7F 32
				   &anatop IMX_CCM_AUDIO_POST_DIV_SEL 0x70 0x18 1
				   &anatop IMX_CCM_ANALOG_PLL_AUDIO_NUM 0x80 0x3FFFFFFF 77
				   &anatop IMX_CCM_ANALOG_PLL_AUDIO_DENOM 0x90 0x3FFFFFFF 100>;
```
I need provide a name for a clock setting, the defined value is select at will. just to provide a clock name

Reply:
I think this can deleted from the dts as well as it is not used by the clock driver.

Reply:
it is used in dts for audio codec setting
```
		pd = DT_PHA_BY_IDX(DT_DRV_INST(0), pll_clocks, 2, value);
		num = DT_PHA_BY_IDX(DT_DRV_INST(0), pll_clocks, 3, value);
		den = DT_PHA_BY_IDX(DT_DRV_INST(0), pll_clocks, 4, value);
...

	      audioPllConfig.loopDivider = lp;
	      audioPllConfig.postDivider = pd;
	      audioPllConfig.numerator = num;
	      audioPllConfig.denominator = den;
	      audioPllConfig.src = src;
      
	      CLOCK_InitAudioPll(&audioPllConfig);
```

Reply:
remove those and use a simple list, updated